be/src/storage/table_reader.cpp

```diff
@@ -264,6 +264,7 @@ Status TableReader::_tablet_multi_get_rpc(doris::PBackendService_Stub* stub, int
         closure->cntl.set_timeout_ms(_params->timeout_ms);
     }
     stub->local_tablet_reader_multi_get(&closure->cntl, &request, &closure->result, closure);
+    closure->ref();
```

Comment:
This closure is `referenced` in 256, which means this closure can not be deleted in this function.
So why do you add a new ref here?

Reply:
@imay 
At 
 [ref_count_closure.h#L41](https://github.com/StarRocks/starrocks/blob/45f45d98e1be6f1c0f449047bd759c3428983c26/be/src/util/ref_count_closure.h#L41), will count down ref count when rpc return, ater `closure->join()`, 'closure' was deleted.
Expected behavior is delete at end of function, ref twice, callback unref once, end of function unref and delete.